app/src/main/res/xml/player_settings.xml

```diff
@@ -78,6 +79,13 @@
             app:title="@string/fullscreen_orientation"
             app:useSimpleSummaryProvider="true" />
 
+        <SwitchPreferenceCompat
```

Comment:
I think we don't need this preference (we already have quite a lot of player settings which makes it very hard to overview for users), it's okay to always ask to play the downloaded version when a downloaded version has been found.